core/src/main/java/com/klaytn/caver/utils/AccountKeyPublicUtils.java

```diff
@@ -51,8 +51,17 @@ public static AccountKeyPublic decompressKey(String compressedPublicKey) {
         compEnc[0] = (byte)(yBit ? 0x03 : 0x02);
         ECPoint ecPoint = CURVE.getCurve().decodePoint(compEnc);
         return AccountKeyPublic.create(
-                Numeric.prependHexPrefix(ecPoint.getAffineXCoord().toString()),
-                Numeric.prependHexPrefix(ecPoint.getAffineYCoord().toString())
+                Numeric.prependHexPrefix(leftPaddingZeroTo64Length(ecPoint.getAffineXCoord().toString())),
+                Numeric.prependHexPrefix(leftPaddingZeroTo64Length(ecPoint.getAffineYCoord().toString()))
```

Comment:
Can we use `toHexStringWithPrefixZeroPadded` instead of implementing a new function for this?

Reply:
I feel same way with @kjhman21. Without creating new method, you can use like `Numeric.toHexStringWithPrefixZeroPadded(ecPoint.getAffineXCoord().toBigInteger(), 64)`.

Reply:
Thank you for knowing something good. I will reflect the review.